scriptmodules/ports/alephone.sh

```diff
@@ -54,18 +54,22 @@ function install_alephone() {
 }
 
 function game_data_alephone() {
-    local release_url="https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620"
+    local version="20150620"
+    if compareVersions "$__os_debian_ver" ge 9 || [[ -n "$__os_ubuntu_ver" ]]; then
+        local version="20220115"
```

Comment:
This doesn't need the additional "local".

Reply:
You're right, of course. Thanks.